test/clj/rems/test/sendmail.clj

```diff
@@ -46,8 +46,7 @@
   (let [common-name "Test User"
         applicant-attrs {"eppn" "test-user" "mail" "invalid-addr" "commonName" common-name}]
```

Comment:
`"test-user"` is used in lots of places in this test, so it would be good to extract to a variable, e.g. `applicant-id`